lib/api/controller/auth.js

```diff
@@ -188,9 +189,74 @@ class AuthController extends NativeController {
         request.context.token);
     }
 
+    if ( global.kuzzle.config.security.supportCookieAuthentication
+      && request.input.args.cookieOnly
+    ) {
+      request.setResult(null,
+        {
+          headers: {
+            'Set-Cookie': Cookie.serialize(
+              '__Host-authToken',
+              '',
+              {
+                httpOnly: true,
+                path: '/',
+                sameSite:'none',
+                secure: true,
+              }
+            )
+          }
+        });
+    }
+
     return {acknowledged: true};
   }
 
+  // Used to send the Token using different ways when in cookieOnly mode. (DRY)
+  async _sendToken(token, request) {
+    // Only if the support of Browser Cookie as Jwt is enabled
+    // otherwise we should send a normal response because
+    // even if the SDK / Browser can handle the cookie,
+    // Kuzzle would not be capable of doing anything with it
+    if (global.kuzzle.config.security.supportCookieAuthentication
+      && request.input.args.cookieOnly
+    ) {
+      // Here we are not sending jwt when cookieOnly is set to true
+      // This allow us to detect if kuzzle does support cookie as jwt directly from the SDK
+      // if the response contains the jwt this means that the configuration [security.supportCookieAuthentication] is set to false
+      // or that the version of kuzzle doesn't support the feature Browser Cookie as Jwt
```

Comment:
Nitpicking: domain wording
```suggestion
      // Here we are not sending auth token when cookieOnly is set to true
      // This allow us to detect if kuzzle does support cookie as auth token directly from the SDK
      // if the response contains the auth token this means that the configuration [security.supportCookieAuthentication] is set to false
      // or that the version of kuzzle doesn't support the feature Browser Cookie as Authentication Token
```